CMakeLists.txt

```diff
@@ -111,6 +111,18 @@ if (USE_CCACHE)
     endif()
 endif()
 
+#################################
+# Scafacos Package (optional)   #
+#################################
+
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(SCAFACOS IMPORTED_TARGET scafacos)
+
+if ( SCAFACOS_FOUND )
+  include_directories("${SCAFACOS_INCLUDEDIR}")
+  add_definitions( -DHAVE_CONFIG_H )
+endif()
```

Comment:
Please do not change global properties. Add the corresponding targets when needed.

Reply:
Yeah I think as you are using `PkgConfig::SCAFACOS` below, just remove the whole block.

Plus, why do we need `-DHAVE_CONFIG_H`?

Reply:
HAVE_CONFIG_H is a flag used for compiling scafacos_interface. Perhaps it is not anymore necessary to add and I will try a compilation w/o it to see whether it passes.

Reply:
Update: `-DHAVE_CONFIG_H` is a must to have. Otherwise, scafacos will crash in force calculations.

Reply:
It is used in a C directive and to read activation flags for selected solvers and (/or) internal (math) libraries.

> `#ifdef HAVE_CONFIG_H`
> `#include <config.h>`
> `#endif`